setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='django-model-changes-py3',
-    version='0.15.1',
```

Comment:
This version bump necessary so the cached package isn't used during pip install.